frappe/commands/site.py

```diff
@@ -331,7 +332,14 @@ def drop_site(site, root_login='root', root_password=None, archived_sites_path=N
 
 	frappe.init(site=site)
 	frappe.connect()
-	scheduled_backup(ignore_files=False, force=True)
+
+	try:
+		scheduled_backup(ignore_files=False, force=True)
+	except ProgrammingError as err:
+		if err[0] == 1146:
```

Comment:
what is the root cause? best to put in a prompt asking that the backup has failed, does the user want to continue?